catalog/spatial/registry/registry-federation-admin-service-impl/pom.xml

```diff
@@ -81,6 +81,11 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-ext</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.karaf.shell</groupId>
+            <artifactId>org.apache.karaf.shell.core</artifactId>
+            <version>4.0.4</version>
```

Comment:
can you update to use ${karaf.version} here? It will make this dependency easier to maintain.